UI/Components/PossibleTimeSaveSettings.cs

```diff
@@ -89,6 +92,7 @@ void PossibleTimeSaveSettings_Load(object sender, EventArgs e)
             rdoSeconds.Checked = Accuracy == TimeAccuracy.Seconds;
             rdoTenths.Checked = Accuracy == TimeAccuracy.Tenths;
             rdoHundredths.Checked = Accuracy == TimeAccuracy.Hundredths;
+            chkDropDecimals.Checked = DropDecimals;
```

Comment:
I don't think this is necessary either, but please test and verify.

Reply:
Verified unnecessary, removed in next commit